README.md

```diff
@@ -4,15 +4,15 @@ __Solve the task using OOP principles__
 
 Task:
 
-There are some figures of the following types: square, rectangle, right triangle, circle, isosceles trapezoid.
+There are some figure of the following types: square, rectangle, right triangle, circle, isosceles trapezoid.
```

Comment:
You have several figures, so it's plural, and we should plural form of the word `figure`